speed-dreams/source-2.2.3/data/cars/models/primo/primo.xml

```diff
@@ -184,7 +184,7 @@
 		<attnum name="revs maxi" unit="rpm" val="10000"/>
 		<attnum name="revs limiter" unit="rpm" val="4500"/>
 		<attnum name="tickover" unit="rpm" val="100"/>
-		<attnum name="fuel cons factor" val="1.1"/>
+		<attnum name="fuel cons factor" val="0"/>
```

Comment:
Is this line still needed since you removed the code that loads it

Reply:
I am not going to remove this one line, because otherwise it would be more logical to remove it for more files which is not what we wanted. Besides, this makes it easier for future developers to change the code anyways.

Reply:
Right now it's confusing, because setting it to a value doesn't do anything while users will expect it to have an effect

Reply:
Alright if we remove the fuel cons factor from primo.xml then?

Reply:
Yes that's fine, just replace it with a comment that states that it was removed.